### PR TITLE
Add stacktable.js@1.0.1 w/ git autoupdate.

### LINKED ISSUE #9486: [Request] Add stacktable
**Library name:** stacktable
**Git repository url:** https://github.com/johnpolacek/stacktable.js/
**License(s):** Dual MIT & GPL license
**Official homepage:** http://johnpolacek.github.io/stacktable.js/

### PR DESCRIPTION
**Pull request by Piicksarn**
@cdnjs/intern @cdnjs/intern2, 
I need your help for reviewing the pr.
Thank you all!
Pull request for issue: #9486.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/johnpolacek/stacktable.js/
 * Official website (optional, not the repository): http://johnpolacek.github.com/stacktable.js/
 * NPM package url (optional): - 
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 47
   - Count of watchers: 850
   - Count of forks: 116
 * NPM download stats (optional): -
   - Downloads in the last day: -
   - Downloads in the last week: -
   - Downloads in the last month: -

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/stacktable.js/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "license": "MIT",
+  "homepage": "http://johnpolacek.github.com/stacktable.js/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/johnpolacek/stacktable.js.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "stacktable*.+(css|js)"
```
Comment (maruilian11) [inline ajax/libs/stacktable.js/package.json]: png?

File: ajax/libs/stacktable.js/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    "js",
+    "javascript",
+    "jquery",
+    "tables"
+  ],
+  "author": {
+    "name": "John Polacek",
+    "url": "http://www.johnpolacek.com"
+  },
+  "license": "MIT",
```
Comment (maruilian11) [inline ajax/libs/stacktable.js/package.json]: GPL?
https://github.com/johnpolacek/stacktable.js#legal

**Comment by maruilian11** [comment]
we can ask the author if he can also update npm package and add minified files.

File: ajax/libs/stacktable.js/package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  "homepage": "http://johnpolacek.github.com/stacktable.js/",
```
Comment (x09326) [inline ajax/libs/stacktable.js/package.json]: @Piicksarn You can use `https` instead of `http` here.

Comment (pvnr0082t) [inline ajax/libs/stacktable.js/package.json]: https://johnpolacek.github.com/stacktable.js/ will also be redirect to http://johnpolacek.github.io/stacktable.js/.
I think https://johnpolacek.github.io/stacktable.js/ is better.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by PeterDaveHello** [comment]
Close per #11472

**Pull request closed without merging**